### PR TITLE
Donations summary page for events, add donation message, first donation mailer

### PR DESCRIPTION
**Pull request by tmb**
image

Same as before, not totally sure where you want the `donation.message` field to go so I left it out

### CODE REVIEW

**Comment by lachlanjc** [comment]
If you’re expecting them to be somewhat long sometimes you could put them in a little popover or modal

File: app/controllers/events_controller.rb
Comment (thesephist) [inline app/controllers/events_controller.rb]: I think it makes more sense for this list to be reverse-chronological, so it matches the transactions list.

File: app/mailers/donation_mailer.rb
Comment (thesephist) [inline app/mailers/donation_mailer.rb]: This seems a bit long and might not show on phone screens in one line.. how about

>Congrats on receiving your first donation! 🎉

Comment (thesephist) [reply app/mailers/donation_mailer.rb]: Specifying "first" may also clear up confusion, so people are less likely to get the impression that they'll get notified for every donation.

File: app/models/donation.rb
Comment (thesephist) [inline app/models/donation.rb]: Might we want to raise an exception here so we (devs) are notified? This should never happen in practice, right? So it's better to fail loudly here since it probably means we messed up some code logic somewhere and need to fix it.

File: app/views/donation_mailer/first_donation_notification.html.erb
Comment (thesephist) [inline app/views/donation_mailer/first_donation_notification.html.erb]: Might be nice to include the first donation's donor and amount, so it's not super generic. Maybe like:

>Congrats! We're emailing to let you know that you've received your first-ever donation of $XX from $XX.

File: app/views/events/donation_overview.html.erb
Comment (thesephist) [inline app/views/events/donation_overview.html.erb]: I think this UI needs work but we can handle that once we merge to master.

File: app/views/events/_nav.html.erb
Comment (thesephist) [inline app/views/events/_nav.html.erb]: Can we put this behind the `donations_page_enabled` flag?

File: db/schema.rb
Comment (thesephist) [inline db/schema.rb]: Is it just me or does this PR not include a migration to add these two columns?

**Review by thesephist — CHANGES_REQUESTED** [review]
TLDR almost ready to merge, we can polish UI stuff once it's in master.

File: app/models/donation.rb
Comment (tmb) [reply app/models/donation.rb]: If we raise an exception the whole view will break (bad)—I'm leaving Unknown there with the hope that people will hit us up.

File: db/schema.rb
Comment (tmb) [reply db/schema.rb]: Another one of the donations one does that we already merged.

**Pull request merged**
